typing/typecore.ml

```diff
@@ -5701,7 +5713,9 @@ and type_expect_
         in
         match expected_opath, opt_exp_opath with
         | None, None ->
-          newvar (Jkind.of_new_sort ~why:Record_projection), None
+          newvar
+            (Jkind.of_new_sort ~why:Record_projection
+               ~level:(Ctype.get_current_level ())), None
```

Comment:
I think `None` should probably go on its own line here, to make the pair structure clear.